tox.ini

```diff
@@ -1,25 +1,37 @@
 [tox]
 envlist =
-    py27-django111
-    py{34,35,36}-django{111,20}
-    pypy2-django111
-    pypy3-django{111,20}
+    py{27,34,35,36}-django111
+    py{35,36,37}-django{20,21}
+    py38-django22
     lint
 
+[tox:travis]
+2.7 = py27
+3.4 = py34
+3.5 = py35
+3.6 = py36
+3.7 = py37
+3.8 = py38
+
 toxworkdir = {env:TOX_WORKDIR:.tox}
 
 [testenv]
 deps =
     -rrequirements_test.txt
+    coverage
     django111: Django>=1.11,<1.12
     django20: Django>=2.0,<2.1
+    django21: Django>=2.1,<2.2
+    django22: Django>=2.2,<3.0
 
 whitelist_externals = make
-commands = make test
```

Comment:
Maybe remove the `test` target of the Makefile? It seems not correct anymore.

Reply:
Good catch ! Done :)